src/llvm.cpp

```diff
@@ -553,6 +553,15 @@ llvm::Value *emitExprSelector(Context *ctx, Expr *expr) {
             return ctx->b.CreateAlignedLoad(addr, BytesFromBits(info.value.Struct.index));
         }
 
+        case SelectorKind_Enum: {
+            llvm::Type *type = canonicalize(ctx, info.type);
+            if (ctx->returnAddress) {
+                UNIMPLEMENTED();
```

Comment:
The frontend should report errors if a user tries to address an enum. This should be an `ASSERT(false)`